cmd/main.go

```diff
@@ -210,3 +172,52 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func bindConfig(opts zap.Options, cfg operator.Config) {
```

Comment:
Also pass *FlagSet here, then we can control the input

Reply:
I've made an attempt, but my original plan was to refactor this when trying to build tests. I have no idea if this will be testable because of the `pflag.CommandLine` used in the func.

Reply:
we could either add another testable function that takes `pflag.CommandLine` as input or replace `flag` with `pflag`.